src/simulations/set_generator.py

```diff
@@ -97,14 +115,14 @@ def f(random_state):
 
     return f
 
-  def __init__(self, universe_size, set_sizes, random_state):
+  def __init__(self, universe_size, set_size_list, random_state):
```

Comment:
I'd continue to call this "set_sizes" since we generally expect an iterable not a list per se.

Reply:
Done.